controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/aws/CloudEvent.java

```diff
@@ -0,0 +1,30 @@
+package com.yahoo.vespa.hosted.controller.api.integration.aws;
+
+import java.util.Date;
+import java.util.Optional;
+import java.util.Set;
+
+public class CloudEvent {
+    public String instanceEventId;
+    public String code;
+    public String description;
+    public Optional<Date> notBefore;
+    public Optional<Date> notBeforeDeadline;
+    public Optional<Date> notAfter;
+
+    public String regionName;
+    public Set<String> affectedHostnames;
+
+    public CloudEvent(String instanceEventId, String code, String description, Date notAfter, Date notBefore, Date notBeforeDeadline,
+                      String regionName, Set<String> affectedHostnames) {
```

Comment:
This should be `awsRegionName` (not to be confused with hosted Vespa region name)